packages/api/src/middleware/errorHandler.ts

```
@@ -0,0 +1,113 @@
+        ...(error instanceof Error && error.stack && { stack: error.stack }),
+      };
+
+      if (logLevel === "error") {
+        logger.error("Request failed with error", logContext);
+      } else {
+        logger.warn("Request failed", logContext);
+      }
+
+      // Send standardized error response
+      const response: ApiErrorResponse = apiError.toResponse(requestId);
+      return c.json(response, { status: apiError.statusCode });
```

ESLint

[error] Async arrow function expected no return value.

(consistent-return)